extension/g3kb-switch@g3kb-switch.org/metadata.json

```diff
@@ -10,6 +10,10 @@
         "44"
     ],
     "url": "https://github.com/lyokha/g3kb-switch",
-    "version" : 1
+    "version" : 1,
+    "session-modes" : [
+        "user",
+        "unlock-dialog"
+  ]
```

Comment:
Please align this brace with other items (to keep 4-spaces indentation).

Reply:
maybe I'm missing something, but this has proper indentation, same as the key `shell-version`

Reply:
```json
    "shell-version": [
        "40",
        "41",
        "42",
        "43",
        "44"
    ],
    "url": "https://github.com/lyokha/g3kb-switch",
    "version" : 1,
    "session-modes" : [
        "user",
        "unlock-dialog"
  ]
```

to

```json
    "shell-version": [
        "40",
        "41",
        "42",
        "43",
        "44"
    ],
    "url": "https://github.com/lyokha/g3kb-switch",
    "version" : 1,
    "session-modes" : [
        "user",
        "unlock-dialog"
    ]
```

Note that now the closing brace is aligned with starting of top items and their closing braces, see that now it's on the same column with the closing bracket of `"shell-version"`.

Reply:
right sorry, the closing bracket.
I thought you meant the content of the array.
fixed